test/regression.uts

```diff
@@ -6879,3 +6879,66 @@ x = f.i2repr(mp, {'*', '+', 'bit 2'})
 assert(re.match(r'^.*Star \(\*\).*$', x) is not None)
 assert(re.match(r'^.*Plus \(\+\).*$', x) is not None)
 assert(re.match(r'^.*bit 2.*$', x) is not None)
+
+###########################################################################################################
```

Comment:
Sorry I missed that. You're right. But maybe you could double the tests? Like that:
```Python
compressed1, compressed2 = _ipv6_bin_to_str(address), inet_ntop(socket.AF_INET6, address)
assert compressed1 == compressed2 == '::'
```
What do you think?